sway-lib-std/src/math.sw

```diff
@@ -193,15 +211,65 @@ impl BinaryLogarithm for u8 {
     }
 }
 
+impl BinaryLogarithm for u256 {
+    /// Calculates the binary log.
```

Comment:
Similarly to previous comment regarding documentation, this documentation can be removed